homeassistant/components/atome/sensor.py

```diff
@@ -251,16 +251,6 @@ def unit_of_measurement(self):
         """Return the unit of measurement."""
         return self._unit_of_measurement
 
-    @property
-    def icon(self):
```

Comment:
You're not setting the `_attr` for this one.

Reply:
oh it has a device class with that icon. 👍

Reply:
Yeah, and the icon used is the default icon for the device class :)